hudi-common/src/test/java/org/apache/hudi/common/properties/TestTypedProperties.java

```diff
@@ -99,8 +99,37 @@ public void testPropertiesOrder() throws IOException {
     properties.put("key9", "true");
 
     TypedProperties typedProperties = new TypedProperties(properties);
+    assertTypeProperties(typedProperties, 0);
+  }
+
+  @Test
+  void testPutAllProperties() {
```

Comment:
is this test failing w/o the fix in this patch ?

Reply:
it still works with Java 8, but will fail with Java 11.